ci/run_envoy_docker.sh

```diff
@@ -75,6 +75,7 @@ docker run --rm \
        -e BAZEL_EXTRA_TEST_OPTIONS \
        -e BAZEL_REMOTE_CACHE \
        -e ENVOY_STDLIB \
+       -e LLVM_ROOT \
```

Comment:
This is the env that I do not want to leak into CI containers, you should be able to specify it like `ci/run_envoy_docker.sh 'LLVM_ROOT=<alternative LLVM path in your image> ci/do_ci.sh bazel.release'`, or if your build image have different path you can build env var into your image.

I usually have LLVM_ROOT set in my environment outside of docker, (e.g. `/usr/lib/llvm-11` in debian), though when I do run CI script this shouldn't be passed into CI image.

Reply:
I can get that. If what you specified doesn't work, I have a couple ideas of what we can do.

Reply:
Turns out it works, not sure why I didn't think to do this prior